### PR TITLE
Integration with configuration service

### PR DESCRIPTION
**Pull request by 95kach**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
   },
   "author": "Stephane Benayoun (https://github.com/stephanebenayoun)",
   "license": "MIT",
-  "homepage": "https://github.com/capsulajs/account-service#readme",
   "dependencies": {
+    "@capsulajs/capsulajs-configuration-service": "^0.1.2",
```
Comment (MikeSquall) [inline package.json]: you can upgrade to 0.1.3 with `lib/` built inside

File: src/providers/AccountService.js [hunk 2]
```diff
@@ -25,67 +25,56 @@ export class AccountService implements AccountServiceInterface {
     this.organizationService = new OrganizationService(dispatcher);
     this.token = token;
   }
+  
+  prepRequest(request) {
+    return {
+      token: {
+        token: this.token,
+        issuer: 'Auth0'
```
Comment (MikeSquall) [inline src/providers/AccountService.js:33]: Shouldn't we have this value in some config file ? Will it be `Auth0` forever or could this change ?

Comment (95kach) [reply src/providers/AccountService.js:33]: It can be changed in future according to the backend API changes

File: package.json (hunk 1 above)
Comment (95kach) [reply package.json]: sure

**Pull request merged**